### ISSUE TITLE
"Rebuild Current Project" action when using FileSystemWorkspace ?

### ISSUE THREAD

**Issue by uffejakobsen** [eranif/codelite#2722]
When using FileSystemWorkspace the only build targets available in the FSW workspace settings are "build" and "clean"

"Rebuild Current Project" is active in the Build menu - but does not seem to do any thing.

How do I assign an action to "Rebuild Current Project" when using FileSystemWorkspace ?

**Comment by eranif** [comment]
You can add as many targets as you want to the FSW
Open the `settings` -> `build` tab
and click on the `New...` button

Add new target `rebuid` and set the command
This entry is now available from the drop down menu 

![build](https://user-images.githubusercontent.com/3677711/116565033-26c4cc80-a90e-11eb-81b0-cf515a788c80.png)

**Comment by uffejakobsen** [comment]
Thanks - I was looking for at way to trigger the targets through a shortcut - having to use the mouse on the dropdown list 100's of times every day is not really desireable :-)

That was why I was asking for a way to relate the actions in the "Build" menu to the targets in the FSW settings
But I guess that is not possible at the moment ?

**Comment by eranif** [comment]
Nope. Lets keep this bug opened (I will add the `important` label to it so the stale bot will skip it)
